src/Symfony/Component/Serializer/Tests/NameConverter/XmlAttributesConverterTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Serializer\Tests\NameConverter;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\NameConverter\XmlAttributesConverter;
+
+class XmlAttributesConverterTest extends TestCase
+{
+    /**
+     * @dataProvider normalizeDataProvider
+     */
+    public function testNormalize(
+        ?string $attributePrefix,
+        ?string $nodeValueAttributeName,
+        string $propertyName,
+        string $expectedPropertyName
+    ): void {
+        $xmlAttributeConverter = $this->createXmlAttributesConverter($attributePrefix, $nodeValueAttributeName);
+        $result = $xmlAttributeConverter->normalize($propertyName);
+        $this->assertEquals($expectedPropertyName, $result);
+    }
+
+    /**
+     * @dataProvider denormalizeDataProvider
+     */
+    public function testDenormalize(
+        ?string $attributePrefix,
+        ?string $nodeValueAttributeName,
+        string $propertyName,
+        string $expectedPropertyName): void
```

Comment:
Void needs to be on the next line

Reply:
looks like fabbot.io doesn't like it.

Reply:
Ah damn the use long lines for method signatures.
Sorry to many projects 😊

Reply:
Thanks for the review!